selenium.go

```diff
@@ -330,6 +363,44 @@ type WebDriver interface {
 	// ButtonUp causes the left mouse button to be released.
 	ButtonUp() error
 
+	// KeyPauseAction build a KeyAction which pauses for the supplied duration
+	KeyPauseAction(duration time.Duration) KeyAction
+	// KeyUpAction build a KeyAction press
+	KeyUpAction(key string) KeyAction
+	// KeyDownAction build a KeyAction which presses and holds the specified
+	// key until explicitly released by KeyUpAction or ReleaseActions
+	KeyDownAction(key string) KeyAction
+	// StoreKeyActions
+	//
+	// inputId: unique device identifier for the stored action
+	//
+	// actions: any number of KeyActions
+	StoreKeyActions(inputId string, actions ...KeyAction)
+
+	// PointerPause build a PointerAction which pauses for the supplied duration
+	PointerPauseAction(duration time.Duration) PointerAction
+	// PointerMove build a PointerAction which moves the pointer
+	PointerMoveAction(duration time.Duration, offset Point, origin PointerMoveOrigin) PointerAction
+	// PointerUp build an action which releases the specified pointer key
+	PointerUpAction(button MouseButton) PointerAction
+	// PointerDown build a PointerAction which presses and holds the specified
+	// pointer key until explicitly released by PointerUp or ReleaseActions
+	PointerDownAction(button MouseButton) PointerAction
```

Comment:
None of these "...Action" methods need to be methods, as they don't require the WebDriver instance.

Either:

1) Move them into their own package called `package action` so that the references can be, for example, `action.PointerPause` or `action.PointerDown`.

2) Move them to be free functions in this package.

The benefit of #1 follows from https://github.com/golang/go/wiki/CodeReviewComments#package-names . The benefit of #2 is that the documentation for these actions is contained onto a single page. (Still, for #1, the package will be linked.

I prefer #1, but I'm open to suggestions.